wordpress/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM clearlinux/php-fpm
+MAINTAINER sophia.gong@intel.com
+
+ARG swupd_args
+
+RUN swupd update $swupd_args && \
```

Comment:
It doesn't need do "update" again because it bases on the updated clearlinux/php-fpm.
So please remove these lines.

Reply:
Modified and re-submit at https://github.com/clearlinux/dockerfiles/pull/178. please take a review.

Reply:
update mariadb bind address at https://github.com/clearlinux/dockerfiles/pull/179.